libs/sdk-model/src/execution/buckets/index.ts

```diff
@@ -423,3 +424,26 @@ export function disableComputeRatio<T extends IAttributeOrMeasure>(item: T): T {
 
     return item;
 }
+
+/**
+ * Describes the type of the function used to modify the bucket items.
+ *
+ * @public
+ */
+export type BucketItemModifier = (bucketItem: IAttributeOrMeasure) => IAttributeOrMeasure;
+
+/**
+ * Creates a new bucket with modified bucket items (retrieved by applying the modifyFunction to the bucketItem).
```

Comment:
I suggest to reword this as follows:

`Creates a new bucket by modifying items of the provided input bucket. Each item from the input bucket will be dispatched to the modification function and the result of the modification will be included in the new bucket.

Note: it is valid for the modification function to just return the original item. In that case the item will be included in the bucket without modification.
`

Reply:
Thanks, I've updated the PR.